frontend/src/libraries/explorer-wamp/stats.ts

```diff
@@ -0,0 +1,39 @@
+import { ExplorerApi } from ".";
+
+export interface TransactionsByDate {
+  date: string;
+  transactions: number;
```

Comment:
```suggestion
  transactionsCount: number;
```

Apply the same naming for accounts and contracts